client/analysis/output.go

```diff
@@ -275,65 +180,39 @@ func PrintBrakemanOutput(mongoDBcontainerOutput string, mongoDBcontainerInfo str
 	}
 
 	for _, warning := range brakemanOutput.Warnings {
-		if warning.Confidence == "High" {
-			foundVuln = true
-			color.Red("[HUSKYCI][!] Confidence: %s", warning.Confidence)
-			color.Red("[HUSKYCI][!] Type: %s", warning.Type)
-			color.Red("[HUSKYCI][!] Details: %s", warning.Details)
-			color.Red("[HUSKYCI][!] Info: %s", warning.Message)
-			color.Red("[HUSKYCI][!] File: %s", warning.File)
-			color.Red("[HUSKYCI][!] line: %d", warning.Line)
-			color.Red("[HUSKYCI][!] Code: %s", warning.Code)
-			fmt.Println()
-		}
-
-		if warning.Confidence == "Medium" {
-			foundVuln = true
-			color.Yellow("[HUSKYCI][!] Confidence: %s", warning.Confidence)
-			color.Yellow("[HUSKYCI][!] Type: %s", warning.Type)
-			color.Yellow("[HUSKYCI][!] Details: %s", warning.Details)
-			color.Yellow("[HUSKYCI][!] Info: %s", warning.Message)
-			color.Yellow("[HUSKYCI][!] File: %s", warning.File)
-			color.Yellow("[HUSKYCI][!] line: %d", warning.Line)
-			color.Yellow("[HUSKYCI][!] Code: %s", warning.Code)
-			fmt.Println()
-		}
-
-		if warning.Confidence == "Low" {
-			foundInfo = true
-			color.Blue("[HUSKYCI][!] Confidence: %s", warning.Confidence)
-			color.Blue("[HUSKYCI][!] Type: %s", warning.Type)
-			color.Blue("[HUSKYCI][!] Details: %s", warning.Details)
-			color.Blue("[HUSKYCI][!] Info: %s", warning.Message)
-			color.Blue("[HUSKYCI][!] File: %s", warning.File)
-			color.Blue("[HUSKYCI][!] line: %d", warning.Line)
-			color.Blue("[HUSKYCI][!] Code: %s", warning.Code)
-			fmt.Println()
-		}
+		brakemanVuln := types.HuskyCIVulnerability{}
+		brakemanVuln.SecurityTool = "brakeman"
+		brakemanVuln.Confidence = warning.Confidence
+		brakemanVuln.Details = warning.Details + warning.Message
+		brakemanVuln.File = warning.File
+		brakemanVuln.Line = strconv.Itoa(warning.Line)
+		brakemanVuln.Code = warning.Code
+		brakemanVuln.Type = warning.Type
+
+		rubyResults.BrakemanOutput = append(rubyResults.BrakemanOutput, brakemanVuln)
 	}
 
 	if foundVuln {
-		color.Red("[HUSKYCI][X] Brakeman :(\n\n")
 		types.FoundVuln = true
-	} else if foundInfo {
-		fmt.Printf("[HUSKYCI][*] Brakeman :|\n\n")
-	} else {
-		color.Green("[HUSKYCI][*] Brakeman :)\n\n")
 	}
-
 }
 
-// PrintSafetyOutput will print Safety output.
-func PrintSafetyOutput(mongoDBcontainerOutput string, mongoDBcontainerInfo string) {
+// PrepareSafetyOutput will prepare Safety output.
+func PrepareSafetyOutput(mongoDBcontainerOutput string, mongoDBcontainerInfo string) {
 
 	if mongoDBcontainerInfo == "No issues found." {
-		color.Green("[HUSKYCI][*] Safety :)\n\n")
 		return
 	}
 
 	if mongoDBcontainerInfo == "Requirements not found or this project uses latest dependencies." {
-		fmt.Printf("[HUSKYCI][*] requirements.txt not found or this project uses latest dependencies.\n")
-		fmt.Printf("[HUSKYCI][*] Safety :|\n\n")
+		safetyVuln := types.HuskyCIVulnerability{}
+		safetyVuln.SecurityTool = "safety"
+		safetyVuln.Severity = "info"
+		safetyVuln.Confidence = "high"
```

Comment:
I would remove this "high" as huskyCI may not be that sure yet about this security test results. Our actual code can't even infer if the `requirements.txt` is not present or the project uses the `latest dependencies`. 😅 

We can work on this soon!

Reply:
You're right! I'll remove that! Thanks, @rafaveira3